### DISCUSSION TITLE
Feature Request: Alias names for readers

### DISCUSSION THREAD

**Discussion by theelderbeever** [duckdb/duckdb#8003] [Ideas]
For starters DuckDB is a pretty amazing tool!

I have been using it to work with parquet tables in s3 recently and have found myself wanting to be able to alias the `read_parquet('s3://some/long/path/to/*.parquet')` to simplify querying. Typically with a static table this can be done with a `CREATE TABLE table AS SELECT * FROM read_parquet('s3://some/long/path/to/*.parquet')`. However, if the table has parquet files being added to it during the duckdb session from some outside source this method fails to pick up the new files (understandably). It would helpful to be able to do something along the lines of `CREATE TABLE table AS ALIAS read_parquet('s3://some/long/path/to/*.parquet')`. Essentially this becomes just a glorified string interpolation but is still quite helpful as we have begun to experiment with placing duckdb behind thin http apis that need to stay up to date.

Maybe this feature already exists?

**Comment by Tishj** [comment]
I think you're looking for a view <https://duckdb.org/docs/sql/statements/create_view>

**Reply by theelderbeever** [reply]
So I am..... Appreciate it! Thanks!
